src/Sensors.cpp

```diff
@@ -239,7 +265,7 @@ bool Sensors::pmPanasonicRead() {
 String Sensors::hwSerialRead() {
     int try_sensor_read = 0;
     String txtMsg = "";
-    while (txtMsg.length() < 32 && try_sensor_read++ < SENSOR_RETRY) {
+    while (txtMsg.length() < lenght_buffer && try_sensor_read++ < SENSOR_RETRY) {
```

Comment:
why lenght_buffer is a global variable? why it isn't pass via parameter?

Reply:
a parameter in hwSerialRead() ?
Something like this hwSerialRead(lenght_buffer) ?

Reply:
yes. We should reduce the global variables usage.

Reply:
done